docs/CODE_OF_CONDUCT

```diff
@@ -0,0 +1,12 @@
+# Code of Conduct
```

Comment:
These files seem to be duplicate of the `.md` files under `docs`?

Reply:
nope